.vscode/launch.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
this file is also personal setting, should not be committed

Reply:
> this file is personal setting, should not be committed

ok, noted. I will remove it.